core/db/curibio/alembic/env.py

```diff
@@ -18,10 +18,12 @@
 config.set_section_option(section, "TABLE_USER_PASS", os.environ.get("TABLE_USER_PASS"))
 config.set_section_option(section, "TABLE_USER_PASS_RO", os.environ.get("TABLE_USER_PASS_RO"))
 config.set_section_option(section, "MANTARRAY_RO_PASS", os.environ.get("MANTARRAY_RO_PASS"))
+config.set_section_option(section, "OPERATORS_RO_PASS", os.environ.get("OPERATORS_RO_PASS"))
```

Comment:
Why is this being added? I don't see it used anywhere

Reply:
I will delete but I thing it is from the merge